src/entities/BlockedAddressEntity.ts

```diff
@@ -0,0 +1,37 @@
+import { BigNumber } from '@0x/utils';
+import { Check, Column, Entity, Index, PrimaryColumn } from 'typeorm';
+
+import { BigNumberTransformer } from './transformers';
+
+export type BlockedAddressConstructorOpts = Pick<BlockedAddressEntity, 'address'> & Partial<BlockedAddressEntity>;
+@Entity({ name: 'blocked_addresses' })
+export class BlockedAddressEntity {
+    @PrimaryColumn({ name: 'address', type: 'varchar' })
+    @Check('address = lower(address)')
+    public address: string;
+
+    @Index()
+    @Column({ name: 'created_at', type: 'timestamptz', default: () => 'now()' })
+    public createdAt!: Date;
+
+    @Column({ name: 'parent', type: 'varchar', nullable: true })
+    public parent: string | null;
+
+    @Column({ name: 'last_seen_nonce', type: 'numeric', nullable: true, transformer: BigNumberTransformer })
```

Comment:
this is set as numeric in the entity definition but bigint in the migration, I'd label this type bigint and just use number since the range won't require BigNumber

Reply:
good catch!